Formula/manticoresearch.rb

```diff
@@ -1,11 +1,22 @@
 class Manticoresearch < Formula
   desc "Open source text search engine"
   homepage "https://www.manticoresearch.com"
-  url "https://github.com/manticoresoftware/manticoresearch/archive/refs/tags/4.2.0.tar.gz"
-  sha256 "6b4af70fcc56b40aa83e606240b237e47e54c0bfbfdd32c47788d59469ef7146"
   license "GPL-2.0-only"
+  # revision is added because manticore provides own tap manticoresoftware/manticore/manticoresearch with
+  # same version, but another formula with own-build icu and another dependencies.
+  revision 1
   version_scheme 1
   head "https://github.com/manticoresoftware/manticoresearch.git", branch: "master"
+  stable do
+    url "https://github.com/manticoresoftware/manticoresearch/archive/refs/tags/5.0.2.tar.gz"
+    sha256 "ca7828a6841ed8bdbc330516f85ad3a85749998f443b9de319cec60e12c64c07"
+  end
```

Comment:
This should not be in a stable block

Reply:
```
brew audit --strict --online manticoresearch
manticoresearch:
  * 7: col 3: `url` (line 7) should be put before `license` (line 4)
  * 7: col 3: `url` should be put inside `stable` block
  * 8: col 3: `sha256` should be put inside `stable` block
Error: 3 problems in 1 formula detected
```

Reply:
I think that's a false positive caused by the first issue